internal/compiler/generator/cpp.rs

```diff
@@ -19,7 +21,44 @@ pub struct Config {
     pub header_include: String,
 }
 
+
+// Check if word is one of C++ keywords
+fn is_cpp_keyword(word: &str) -> bool {
+    static CPP_KEYWORDS: OnceLock<HashSet<&'static str>> = OnceLock::new();
+    let keywords = CPP_KEYWORDS.get_or_init(|| {
+        let keywords: HashSet<&str> = HashSet::from([
```

Comment:
Try to signore fmt with `#[rustfmt::skip]` something like this:
```suggestion
        #[rustfmt::skip]
        let keywords: HashSet<&str> = HashSet::from([
```